### PR TITLE
Add helper function for changing impact of individual filters

### PR DESCRIPTION
**Pull request by mpedrummer**
This pull request fixes issues #37 and #40.  In adding the helper function, I found an error in the code of getFilterData, and fixed that as part of #40

### CODE REVIEW

**Comment by Awnage** [comment]
Good find, and nice work. My only change would be to lowercase "$Filter" to make it consistent with the rest of the FilterCollection.

**Comment by mpedrummer** [comment]
Hey guys - just checking in.  It's the holidays, so I imagine everyone is busy, but I wanted to check and make sure I did things correctly. :)

**Comment by Awnage** [comment]
Very sorry about that Tim. For some reason github never emailed me that you had responded last month. Thanks for keeping us on target!

File: src/Expose/FilterCollection.php [hunk 1]
```diff
@@ -88,6 +92,21 @@ public function setFilterData($data)
...
+     * Alter the impact level of a specific filter id.
+     *
+     * @param integer $filterId
+     * @param integer $impact
+     */
+    public function setFilterImpact($filterId, $impact) {
+        $filter = $this->getFilterData($filterId);
+
+        if($filter === null) {
+            return;
```
Comment (quantumpacket) [inline src/Expose/FilterCollection.php]: Why are we returning here?

Could this not just be written as:

```php
    public function setFilterImpact($filterId, $impact) {
        $filter = $this->getFilterData($filterId);

        if ($filter !== null) {
            $filter->setImpact($impact);
        }
    }
```

Comment (Awnage) [inline src/Expose/FilterCollection.php]: That would be because I trusted github too much with its merging ability...

Comment (Awnage) [inline src/Expose/FilterCollection.php]: @quantumpacket you confused me. Feel free to submit a patch for that once I recommit @mpedrummer's code.

Comment (mpedrummer) [inline src/Expose/FilterCollection.php]: Actually, I think that was me, and not a merge issue.  I prefer to keep exit conditions clear and short.  While in this case, the "non-exit" is quite brief, if it was longer, I find that you have to understand more code to determine where it may have exited, rather than seeing the explicit condition.

It's also easier to see when you've missed paths in the code when doing code coverage (again, with more complex paths, not so much here)

Basically, though, it's just habit and convention for my day job, and I'll change it if it's wrong for Exposé.
